source/val/validate_mesh_shading.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2022 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Validates ray query instructions from SPV_KHR_ray_query
+
+#include "source/opcode.h"
+#include "source/val/instruction.h"
+#include "source/val/validate.h"
+#include "source/val/validation_state.h"
+
+namespace spvtools {
+namespace val {
+
+spv_result_t MeshShadingPass(ValidationState_t& _, const Instruction* inst) {
+  const SpvOp opcode = inst->opcode();
+  switch (opcode) {
+    case SpvOpEmitMeshTasksEXT: {
```

Comment:
Is this missing the execution model limitation for TaskEXT? Noticed because you include the limitation for MeshEXT below.

Reply:
so it was added in https://github.com/KhronosGroup/SPIRV-Tools/pull/4923/files#diff-17d54d370dd151efbd4e9355ad80369fc5847c1ebfbc894b423d9e6607f9701cR1091

Added a comment as it is definitely not obvious and looks like it is missing